docs/game-mechanics/behaviors/basic-attack.rst

```diff
@@ -1,39 +1,87 @@
-Basic Attack (1)
-================
+`Basic Attack (1) <https://github.com/DarkflameUniverse/DarkflameServer/blob/main/dGame/dBehaviors/BasicAttackBehavior.cpp>`_
```

Comment:
Not a fan of linking it this way. I'm fine with adding a link in general for convenience, but it's not like the intent should be that e.g. people on Uchu ever need to open it for comparison.

In some sense to me the point of these docs is not to have code as documentation. That's a personal preference though and I'm open for discussion.

Also a little weird, that this same text links somewhere else when clicked in a table-of-contents vs. the current page. The ususal intuition is that clickable headings "just" introduce a URL fragment identifier to link directly to some section.

Reply:
ah that makes sense.  as far as I have been told other server emulators have been disbanded and are unused and so I assumed the DLU is the only one that exists.  Even if it was though the docs shouldn't require that you need to understand DLU source vs your own, for example.  I've noticed this while looking around the docs since there isn't anything that uses DLU terms (entity, LWOOBJID, LOT etc.).  Will update my PRs to reflect the requested changes.

Reply:
LOT and LWOOBJID are fine to use, we know from scripts, CDClient and other places that those were the original terms. They're not DLU specific either.

Reply:
And entity is a really common term for a spawned object in game engines afaict.

Reply:
We probably shouldn't use Entity considering that we know they used the term LWOGameObject.